app/webhook.go

```diff
@@ -682,6 +682,9 @@ func (a *App) HandleIncomingWebhook(c *request.Context, hookID string, req *mode
 	webhookType := req.Type
 
 	var hook *model.IncomingWebhook
+	if !hook.Enabled {
```

Comment:
This will result in a nil pointer panic. `hook` needs to be initialized on line 692 first

Reply:
by this, do you mean that line 692 should be moved up by/merged into line 684?